sql/mozfun/mobile_search/normalized_app_name/udf.sql

```diff
@@ -0,0 +1,57 @@
+
+
+CREATE OR REPLACE FUNCTION `normalized_app_name`(app_name STRING, os STRING)
```

Comment:
Sorry to be annoying, but I think by convention it should probably be `normalize_app_name` (a verb describing what the function does) rather than `normalized_app_name` (the result of the function)

```suggestion
CREATE OR REPLACE FUNCTION `normalize_app_name`(app_name STRING, os STRING)
```